### PR TITLE
Refractored Shooting Subsystems

### PR DESCRIPTION
**Pull request by imathno**

### CODE REVIEW

File: Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java [hunk 1]
```diff
@@ -14,9 +14,9 @@
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 
-public class SideShooterSubsystem extends BaseSubsystem {
+public class ShooterWheelSubsystem extends BaseSubsystem {
     
-    private static Logger log = Logger.getLogger(ShooterSubsystem.class);
+    private static Logger log = Logger.getLogger(ShooterWheelsManagerSubsystem.class);
```
Comment (JohnGilb) [inline Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java]: This does not match the class name

File: Competition2017/src/competition/subsystems/shooter_wheel/commands/ContinueShooterCommand.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    
+    public void setSide(ShooterWheelSubsystem shooterWheel) {
+        this.shooterWheel = shooterWheel;
+        this.requires(shooterWheel);
+    }
+
+    @Override
+    public void initialize() {
+
```
Comment (JohnGilb) [inline Competition2017/src/competition/subsystems/shooter_wheel/commands/ContinueShooterCommand.java]: There should be a log line here about the command initializing

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: Competition2017/src/competition/subsystems/shooter_wheel/commands/ContinueShooterCommand.java (hunk 2 above)
Comment (aschokking) [reply Competition2017/src/competition/subsystems/shooter_wheel/commands/ContinueShooterCommand.java]: There's another task to refactor these commands completely, I don't think we need to do anything to them about how they currently work

**Review by aschokking — APPROVED** [review]

**Comment by WasabiFan** [comment]
> Small foxes 

🦊

**Pull request merged**